### ISSUE TITLE
Question: How to export table data to csv with quotes?

### ISSUE THREAD

**Issue by hokreb** [jtablesaw/tablesaw#725]
I tried to export table data to csv, so that every data is quotes (I need this due to special characters) by the follow code snipped
```
CsvWriteOptions.Builder builder = CsvWriteOptions.builder(filename)
      .separator(';')
      .header(true)
      .quoteChar('"');
CsvWriteOptions options = builder.build();
data.write().usingOptions(options);
```
But the exported data is not quoted, what do I miss?

**Comment by benmccann** [comment]
You only need to quote a cell if it contains a comma

**Comment by lwhite1** [comment]
this enhancement is implemented in PR#767

**Comment by lwhite1** [comment]
fixed
